ansible/roles_ocp_workloads/ocp4_workload_codeready_workspaces/tasks/workload.yml

```diff
@@ -0,0 +1,116 @@
+---
+# Implement your Workload deployment tasks here
+
+- name: Setting up workload for user
+  debug:
+    msg: "Setting up workload for user ocp_username = {{ ocp_username }}"
+
+- name: Create project for CodeReady Workspaces
+  k8s:
+    state: present
+    definition: "{{ lookup('template', './templates/project.j2' ) | from_yaml }}"
+
+- name: Create Catalogsource for use with catalog mirror
+  when: ocp4_workload_codeready_workspaces_use_catalog_mirror | bool
+  k8s:
+    state: present
+    definition: "{{ lookup('template', item ) | from_yaml }}"
+  loop:
```

Comment:
Why the loop to create the single object?

Reply:
Fine.